homeassistant/components/sensor/dominionenergy.py

```diff
@@ -0,0 +1,49 @@
+"""Support for Washington Dominion Energy."""
+
+from homeassistant.util import Throttle
+from homeassistant.helpers.entity import Entity
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setting the platform in HASS and getting the username and password."""
+    add_devices([DominionEnergySensor(config['username'], config['password'])])
```

Comment:
What happens if the password username combination is wrong? I assume that there will be a traceback. You need to catch the exception and decide what to do. If the username or the password is wrong make the setup fail as otherwise the users will end up with a non-working platform in their instance.

Reply:
do i have to implement this with setup or setup_platform @fabaff ? if there error i will return false 
 if using setup or just provide message to the logger